core/src/test/java/org/apache/jsieve/DiscardTest.java

```diff
@@ -19,72 +19,49 @@
 
 package org.apache.jsieve;
 
-import org.apache.jsieve.exception.SieveException;
+import static org.assertj.core.api.Assertions.assertThat;
+
 import org.apache.jsieve.exception.SyntaxException;
+import org.apache.jsieve.mail.Action;
+import org.apache.jsieve.mail.ActionDiscard;
 import org.apache.jsieve.mail.MailAdapter;
-import org.apache.jsieve.parser.generated.ParseException;
 import org.apache.jsieve.utils.JUnitUtils;
-import org.junit.Assert;
+import org.assertj.core.api.Condition;
 import org.junit.Test;
 
-/**
- * Class DiscardTest
- */
 public class DiscardTest {
 
-    /**
-     * Test for Command 'discard' with invalid arguments
-     */
-    @org.junit.Test
-    public void testInvalidArguments() {
-        boolean isTestPassed = false;
+    private static final Condition<Action> INSTANCE_OF_ACTION_DISCARDED = new Condition<Action>() {
```

Comment:
I can't parse your answer, what do you mean ? guava predicates is java 6, isn't it ?